source/includes/steps-install-mongosync-macos.yaml

```diff
@@ -0,0 +1,31 @@
+stepnum: 1
```

Comment:
[suggestion]

We have started using the `.. procedure::` directive in place of the yaml step files. 

Here's an example from the upgrade docs: https://github.com/10gen/docs-mongodb-internal/pull/1128/files#diff-be567092a426709680a1b9e1e990044a15b2960e934b65f2781f501422b100aeR108 

Here's an example from the downgrade docs: https://github.com/10gen/docs-mongodb-internal/pull/1080/files#diff-a015134d4d03b46f057022780b3bd71eeb89e4a3e3adcdf32ae95e566f240973R119

Reply:
Fixed